app/javascript/packs/hello_react.jsx

```diff
@@ -0,0 +1,26 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
+// like app/views/layouts/application.html.erb. All it does is render <div>Hello React</div> at the bottom
+// of the page.
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+
+const Hello = props => (
+  <div>Hello {props.name}!</div>
+)
+
+Hello.defaultProps = {
+  name: 'David'
+}
+
+Hello.propTypes = {
+  name: PropTypes.string
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  ReactDOM.render(
+    <Hello name="React" />,
+    document.body.appendChild(document.createElement('div')),
+  )
+})
```

Comment:
Out of curiosity are you using this pack anywhere? I couldn't see anywhere it's being used, but that might just me being blind.

Reply:
I believe that pack is installed as part of the react-rails gem - can delete if you wish

Reply:
I see. If it's there as an example to help with building out real packs I think it's fine to leave in for now, but it might be good to overwrite or remove it at some stage.

Reply:
This is something I am interested to learn more about and will reach out outside of this PR thread.